plutus-tx/src/PlutusTx/Ratio.hs

```diff
@@ -94,12 +93,12 @@ instance P.Ord Rational where
   {-# INLINEABLE (>) #-}
   Rational n d > Rational n' d' = (n P.* d') P.> (n' P.* d)
 
-instance Ord Rational where
-  compare (Rational n d) (Rational n' d') = compare (n * d') (n' * d)
-  Rational n d <= Rational n' d' = (n * d') <= (n' * d)
-  Rational n d >= Rational n' d' = (n * d') >= (n' * d)
-  Rational n d < Rational n' d' = (n * d') < (n' * d)
-  Rational n d > Rational n' d' = (n * d') > (n' * d)
+instance Haskell.Ord Rational where
```

Comment:
Just curious: if we can use haskell's `Ord` for plugin, why have `PlutusTx.Ord`? Can you not just use haskell's `Ord` everywhere?